README.md

```diff
@@ -28,6 +28,10 @@ To set up a connection, the driver requires a JDBC connection URL. The connectio
 jdbc:neptune:[connectionType]://[host];[propertyKey1=value1];[propertyKey2=value2]..;[propertyKeyN=valueN]
 ```
 
+A basic example of a connection string is:
+
+```jdbc:neptune:sqlgremlin://neptune-example;port=8182```
```

Comment:
I suggest we make this have `.com` at the end so it is more clear that the neptune-example part is a url, also it lines up with the example provided below.

```suggestion
```jdbc:neptune:sqlgremlin://neptune-example.com;port=8182```
```